net.opentabletdriver.OpenTabletDriver.metainfo.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop-application">
+  <id>net.opentabletdriver.OpenTabletDriver</id>
+  <name>OpenTabletDriver</name>
+  <summary>A cross-platform open-source tablet driver</summary>
+  <developer>
+    <name>InfinityGhost</name>
+  </developer>
+  <metadata_license>GFDL-1.3</metadata_license>
+  <project_license>LGPL-3.0-or-later</project_license>
+  <update_contact>2835365572zty@gmail.com</update_contact>
+  <project_group>OpenTabletDriver</project_group>
+  <description>
+    <p>OpenTabletDriver is an open source, cross platform, user mode tablet driver. The goal of OpenTabletDriver is to be cross platform as possible with the highest compatibility in an easily configurable graphical user interface.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://upload.cc/i1/2024/03/24/9XLkOl.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://upload.cc/i1/2024/03/24/QzaOPw.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://upload.cc/i1/2024/03/24/jk1agN.png</image>
```

Comment:
This should not be temporary image hosting sites, either put them on the website or github repo of upstream